bokehjs/src/lib/core/util/zoom.ts

```diff
@@ -49,9 +48,6 @@ export function scale_range(frame: CartesianFrame, factor: number,
    *   ScaleRange
    */
 
-  // clamp the  magnitude of factor, if it is > 1 bad things happen
-  factor = clamp(factor, -0.9, 0.9)
```

Comment:
This does not seem necessary, and in fact interferes with large scale values for the zoom out tool, which easily end up > 1